widget/edit.html

```diff
@@ -15,86 +15,183 @@ <h3 class="modal-title col-md-9" data-ng-bind="::header"></h3>
+              data-tooltip-append-to-body="true"><i class="icon icon-information font-Size-13"></i></span>
+            <div data-cs-conditional data-ng-if="fieldsArray.length > 0" data-fields="$parent.fields"
+              data-reset-field="$parent.fields" data-mode="'queryFilters'" data-ng-model="$parent.config.query"
+              data-parent-form="editSankeyWidgetForm" name="filterJsonRecord"
+              data-enable-expression="(page==='dashboard' || page==='reporting')" data-show-uuid="true"
+              data-form-name="'editSankeyWidgetForm'">
+            </div>
+            <div data-cs-messages="editSankeyWidgetForm.filterJsonRecord"></div>
+          </div>
+        </div>
+        <div class="alert-danger alert margin-bottom-xlg margin-top-md" data-ng-if="objectFields.length === 0">
+          <span>{{viewWidgetVars.LABEL_SELECT_A_MODULE}}</span>
+        </div>
+      </div>
     </div>
-    <div class="form-group"
-      data-ng-class="{ 'has-error': editSankeyWidgetForm.relationshipNodesField.$invalid && editSankeyWidgetForm.relationshipNodesField.$touched }"
-      data-ng-if="config.relation && params.relationNodePicklistFields">
-      <label for="relationshipNodesField" class="control-label">Relationship Nodes<span
-          class="text-danger">*</span></label>
-      <select name="relationshipNodesField" id="relationshipNodesField" class="form-control"
-        data-ng-options="field.name as field.title for field in params.relationNodePicklistFields"
-        data-ng-model="config.relationshipNodesField" required>
-        <option value="">Select an Picklist</option>
-      </select>
-      <div data-cs-messages="editSankeyWidgetForm.relationshipNodesField"></div>
+    <!-- Live Data -->
+    <div data-ng-if="config.moduleType === 'Across Modules'">
+      <div class="mertics-widget-border padding-top-md padding-bottom-md"></div>
+      <div class="form-group dashed-border-section" style="position:relative;" data-ng-repeat="item in config.layers">
+        <div style="padding: 12px;">
+          <div data-ng-if="$first">
+            <div class="form-group"
+              data-ng-class="{ 'has-error': editSankeyWidgetForm.resource.$invalid && editSankeyWidgetForm.resource.$touched }"
+              data-ng-if="modules">
+              <label for="resource" class="control-label">{{viewWidgetVars.LABEL_RESOURCE}}<span class="text-danger">*</span></label>
+              <select name="resource" id="resource" class="form-control"
+                data-ng-options="module.type as module.name for module in modules" data-ng-model="config.resource"
+                required data-ng-change="changeAttribute()">
+                <option value="">{{viewWidgetVars.LABEL_SELECT_AN_OPTION}}</option>
+              </select>
+              <div data-ng-if="!config.entityTrackable" class="cs-label">{{config.resource | uppercase}} {{viewWidgetVars.TEXT_IS_NOT_TRACKABLE}}</div>
```

Comment:
It should be a single string in the language file, you can pass parameters.
As other languages may have different statement formations.